src/layers/xr-layer/xr-layer.js

```diff
@@ -1,8 +1,8 @@
-import GL from '@luma.gl/constants';
-import { Layer, project32 } from '@deck.gl/core';
-import { Model, Geometry, Texture2D } from '@luma.gl/core';
-import vs from './xr-layer-vertex';
-import fs from './xr-layer-fragment';
+import GL from "@luma.gl/constants";
+import { Layer, project32 } from "@deck.gl/core";
+import { Model, Geometry, Texture2D } from "@luma.gl/core";
+import vs from "./xr-layer-vertex";
+import fs from "./xr-layer-fragment";
```

Comment:
hmm, this branch has `singleQuote: true` in `.prettierrc.yaml`, so I'm not sure why this is toggling.
- I have a slight preference for single-quotes whereever they are valid. (Save so much time not holding down the shift key. 😉 )
- I have a much stronger preference for not having inconsistent style rules getting applied. If we don't address this, diffs are much harder to read, and git-blame will be useless.